[bug] config not working for node_modules including json files

Hi,

I'm using helmet for my angular universal. Including this in my server.ts file breaks my production builds because helmet-csp is trying to require a json file.

TBH, I'm not really good with webpack so I have issues fixing this by myself.


=> Module not found: Error: Can't resolve '../../config'  (config.json is an existing file).


Thanks !

--- a/src/webpack.common.js
+++ b/src/webpack.common.js
@@ -51,7 +51,7 @@ const defaultConfig = function(options, root, settings) {
        *
        * See: http://webpack.github.io/docs/configuration.html#resolve-extensions
        */
-      extensions: ['.ts', '.js'],
+      extensions: ['.ts', '.js', '.json'],
 
       // An array of directory names to be resolved to the current directory
       modules: [

